web/client/src/library/components/plan/Plan.tsx

```diff
@@ -298,42 +271,32 @@ function Plan({
     setPlanAction(EnumPlanAction.Running)
     setPlanState(EnumPlanState.Running)
 
-    debouncedPlanRun({
-      throwOnError: true,
-    })
-      .then(({ data }) => {
-        dispatch([
-          {
-            type: EnumPlanActions.Backfills,
-            backfills: data?.backfills,
-          },
-          {
-            type: EnumPlanActions.Changes,
-            ...data?.changes,
-          },
-          {
-            type: EnumPlanActions.Dates,
-            start: data?.start,
-            end: data?.end,
-          },
-        ])
+    void planRun().then(({ data }) => {
+      dispatch([
+        {
+          type: EnumPlanActions.Backfills,
+          backfills: data?.backfills,
+        },
+        {
+          type: EnumPlanActions.Changes,
+          ...data?.changes,
+        },
+        {
+          type: EnumPlanActions.Dates,
+          start: data?.start,
+          end: data?.end,
+        },
+      ])
 
-        seIsPlanRan(true)
-        setPlanState(EnumPlanState.Init)
+      seIsPlanRan(true)
```

Comment:
Sorry, just noticed this typo.
```suggestion
      setIsPlanRan(true)
```